src/cool_seq_tool/resources/status.py

```diff
@@ -0,0 +1,144 @@
+"""Enable quick status check of Cool-Seq-Tool resources."""
+import logging
+from collections import namedtuple
+from pathlib import Path
+
+from agct._core import ChainfileError
+from asyncpg import InvalidCatalogNameError, UndefinedTableError
+from biocommons.seqrepo import SeqRepo
+
+from cool_seq_tool.handlers.seqrepo_access import SEQREPO_ROOT_DIR, SeqRepoAccess
+from cool_seq_tool.resources.data_files import DataFile, get_data_file
+from cool_seq_tool.sources.uta_database import UTA_DB_URL, UtaDatabase, get_liftover
+
+_logger = logging.getLogger(__name__)
+
+
+ResourceStatus = namedtuple(
+    "ResourceStatus",
+    (
+        "uta",
+        "seqrepo",
+        DataFile.TRANSCRIPT_MAPPINGS.lower(),
+        DataFile.MANE_SUMMARY.lower(),
+        DataFile.LRG_REFSEQGENE.lower(),
+        "liftover",
+    ),
+)
+
+
+async def check_status(
```

Comment:
this method is long -- could be refactored into separate methods for each resource check